serialization/numerics.proto

```diff
@@ -39,3 +39,25 @@ message DoublePrecision {
   required Value value = 1;
   required Error error = 2;
 }
+
+// Polynomials follow.
```

Comment:
Why this comment?

Reply:
We do that in the other protos to indicate that all the subclasses/extensions are grouped together.